### PR TITLE
Beta/review exceptions

### PR DESCRIPTION
**Pull request by tsmith023**

### CODE REVIEW

File: integration/test_collection.py [hunk 1]
```diff
@@ -36,9 +36,9 @@
 from weaviate.collections.classes.tenants import Tenant, TenantActivityStatus
 from weaviate.collections.classes.types import PhoneNumber, WeaviateProperties
 from weaviate.exceptions import (
-    WeaviateQueryException,
+    WeaviateGRPCQueryError,
     WeaviateInsertInvalidPropertyError,
-    WeaviateInsertManyAllFailedError,
+    WeaviateInsertManyAllFailed,
```
Comment (dirkkul) [inline integration/test_collection.py:41]: why did you remove the `Error` here?

File: weaviate/collections/cluster.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from weaviate.collections.classes.cluster import Node, Shards, _ConvertFromREST
 from weaviate.connect import ConnectionV4
 from weaviate.exceptions import (
-    EmptyResponseException,
+    EmptyResponse,
```
Comment (dirkkul) [inline weaviate/collections/cluster.py:8]: also no error?

File: weaviate/collections/config.py [hunk 3]
```diff
@@ -26,8 +26,8 @@
 from weaviate.collections.validator import _raise_invalid_input
 from weaviate.connect import ConnectionV4
 from weaviate.exceptions import (
-    UnexpectedStatusCodeException,
-    ObjectAlreadyExistsException,
+    UnexpectedStatusCodeError,
+    ObjectAlreadyExists,
     WeaviateAddInvalidPropertyError,
```
Comment (dirkkul) [inline weaviate/collections/config.py:31]: error?

File: weaviate/connect/authentication.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
     AuthBearerToken,
     AuthClientCredentials,
 )
-from weaviate.exceptions import MissingScopeException, AuthenticationFailedException
+from weaviate.exceptions import MissingScope, AuthenticationFailed
```
Comment (dirkkul) [inline weaviate/connect/authentication.py:15]: no error?

File: weaviate/exceptions.py [hunk 5]
```diff
@@ -213,17 +278,23 @@ def __init__(self, data: dict):
         super().__init__(msg)
 
 
-class WeaviateGrpcUnavailable(WeaviateBaseError):
+class WeaviateGRPCUnavailable(WeaviateBaseError):
```
Comment (dirkkul) [inline weaviate/exceptions.py:281]: no error?

**Comment by tsmith023** [comment]
Have changed all exceptions to include an `Error` suffix

**Review by dirkkul — APPROVED** [review]
